### ISSUE TITLE
index is being generated blank

### ISSUE THREAD

**Issue by db0** [GDQuest/gdscript-docs-maker#71]
**I'm submitting a...**

- [x] Bug report.

## Bug report ##

**What is the current behavior?**

index.md is being generated empty

**What is the expected behavior?**

index.md should be generated

**Tell us the steps to reproduce the bug, and if possible share a minimal demo of the problem.** 

I am using  `generate_reference.bat -i` to generate the documentation. The index.md is being generated but it's empty.

You can clone [my repo](https://github.com/db0/godot-card-gaming) and generate docs from it to replicate yourselves.

**Comment by NathanLovato** [comment]
Thanks for the reports. Note that right now, I don't have the time at all to work on this repo. Contributions are welcome if you'd like to give it a shot, otherwise, I can't give you an ETA for potential bug fixes.

**Comment by db0** [comment]
It's python right? I'll if I can see something.

**Comment by NathanLovato** [comment]
Yes! Any questions about the code structure, you can ask here or get in touch with me directly on [Discord](https://discord.gg/87NNb3Z)
